### DISCUSSION TITLE
Dodanie pola z identyfikatorem konta Allegro do wątków i wiadomości z Centrum Wiadomości

### DISCUSSION THREAD

**Discussion by hahyes** [allegro/allegro-api#4930] [Suggestions - rejected]
Dzień dobry,

czy byłby problem z dodaniem identyfikatora klienta do wiadomości i wątków pobieranych z Centrum Wiadomości? Chciałbym dosyć szybko identyfikować transakcje klienta po identyfikatorze, który jest w zamówieniach. Niestety nie ma go w wątkach, przez co jestem zmuszony szukać po loginie, co jest dosyć wolne.

Przykład:
```
"interlocutor": {
  "id": "string" <--
  "login": "string",
  "avatarUrl": "string"
}
```

**Comment by MartaNowaczyk** [comment]
Zespół zweryfikował Twoją sugestię i w tym momencie, ze względu na wewnętrzne zasady taka zmiana się nie pojawi.
